### DISCUSSION TITLE
System Identifier

### DISCUSSION THREAD

**Discussion by AidenD5410** [chuot/rdio-scanner#442] [Q&A]
**Id
System identifier in decimal format.**

Where do I find the above information at for my operating system, aswell as for talkgroups

**Comment by SimonSheehan** [comment]
Find the system ID on Radioreference and convert it from hex to decimal.

**Reply by AidenD5410** [reply]
> Find the system ID on Radioreference and convert it from hex to decimal.

I am unable to find the system ID, I am trying to find it for Northampton County, PA. If you can help me find it that’s be great
